src/api/account.ts

```diff
@@ -73,12 +76,7 @@ export abstract class AccountAPI {
     return axios
       .get<IAccountInfoResponse>(url + AccountAPIMethods.INFO + '/' + address)
       .then((response) => response.data)
-      .catch((error) => {
-        if (axios.isAxiosError(error)) {
-          throw new Error('Request error: ' + error.message);
-        }
-        throw error;
-      });
+      .catch((error) => this.isApiError(error));
```

Comment:
Since you're passing the exact same args catch has to `isApiError`, you can just do:

```ts
.catch(this.isApiError)
```